Sofa/Component/Constraint/Lagrangian/Model/src/sofa/component/constraint/lagrangian/model/BilateralInteractionConstraint.inl

```diff
@@ -509,6 +563,33 @@ void BilateralInteractionConstraint<DataTypes>::clear(int reserve)
     }
 }
 
+
+template<class DataTypes>
+Index BilateralInteractionConstraint<DataTypes>::indexOfElemConstraint(int objectId, Index Id)
+{
+    if (objectId == 0)
+    {
+        const SubsetIndices& cIndices = m1.getValue();
+        for (int i = 0; i < cIndices.size(); ++i)
```

Comment:
I suggest you use a range-based for loop, or `std::find`.
Here with you loop, you introduce an implicit conversion (`i < cIndices.size()`) that is avoided with my suggestions

Reply:
thx for the advise. Done and I move the getValue out from the method which is used inside a loop